src/JustEat.StatsD/SupportedTagsFormatter.cs

```diff
@@ -0,0 +1,40 @@
+using JustEat.StatsD.Buffered.Tags;
+
+namespace JustEat.StatsD
+{
+    /// <summary>
+    /// A class that can be used to retrieve supported <see cref="IStatsDTagsFormatter"/> implementations for some of the major metric providers.
+    /// </summary>
+    public static class SupportedTagsFormatter
```

Comment:
Let's rename this to `SupportedTagsFormatters` or `TagFormatters`.

Reply:
I have renamed it to `TagsFormatter` using the singular form to avoid the collision with the namespace `JustEat.StatsD.TagsFormatters`.